blog-service/2023-04-28-apps.md

```diff
@@ -0,0 +1,20 @@
+---
+title: April 27, 2023 (Apps)
```

Comment:
wrong date

Reply:
since the file will be released today, I changed file name and the date in the title as: April 28, 2023